### DISCUSSION TITLE
Is DDD folder structure built-in Nuxt 3 yet?

### DISCUSSION THREAD

**Discussion by Aareksio** [nuxt/nuxt#16373] [Questions]
Hi! Is DDD structure available in Nuxt out of the box?

```
- nuxt.config.js
- modules
  |- blog
    |- pages/
    |- components/
    |- assets/
  |- dashboard
    |- pages/
    |- components/
```

This approach is already possible by using [modules and hooks](https://github.com/Baroshem/nuxt3-structure-modules-pinia/blob/main/modules/blog/module.ts), is it planned to be supported oob with Nuxt 3?

**Comment by TheAlexLichter** [comment]
Yes. But what you want is the `extends` feature! See https://github.com/nuxt/framework/issues/3222 😋
